mediawiki_matrix_bot/__init__.py

```diff
@@ -18,6 +20,25 @@
 log = logging.getLogger("bot")
 logging.basicConfig(level=logging.INFO)
 
+class MLStripper(HTMLParser):
+    def __init__(self) -> None:
+        super().__init__()
+        self.reset()
+        self.strict = False
+        self.convert_charrefs= True
+        self.text = StringIO()
+
+    def handle_data(self, d: str) -> None:
+        self.text.write(d)
+
+    def get_data(self) -> str:
+        return self.text.getvalue()
+
+def strip_tags(html: str) -> str:
```

Comment:
This method is untested. Took it from here: https://stackoverflow.com/questions/753052/strip-html-from-strings-in-python

Reply:
consider it tested :)